license_manager/proto/license_manager.proto

```diff
@@ -126,9 +126,26 @@ message LicenseAllocated {
 }
 
 message RefreshRequest {
-  // Opaque identifier for this invocation, as returned by the server in the
-  // AllocateResponse that allocated this license.
-  string invocation_id = 1; // required
+  // Licenses to refresh.
```

Comment:
uhm ... RefreshRequest and AllocateRequest now are equal? Why not have a single message, then? Semantics also seem pretty much the same?

Reply:
Exploring this in a future PR

Reply:
Should we have 2 different RPCs using the same message, though?

Reply:
Changed to 2 RPCs using the same message